neuromation/api/jobs.py

```diff
@@ -137,6 +137,7 @@ def __init__(self, core: _Core, config: Config, parse: Parser) -> None:
         description: Optional[str] = None,
         is_preemptible: bool = False,
         schedule_timeout: Optional[float] = None,
+        life_span: Optional[float] = None,
```

Comment:
Please document this argument in https://neuromation-sdk.readthedocs.io/en/latest/jobs_reference.html#neuromation.api.Jobs.run

Reply:
sure!